datachannel.go

```diff
@@ -76,11 +76,21 @@ func NewDataChannel(o unsafe.Pointer) *DataChannel {
 	return c
 }
 
+// Send a message over a DataChannel in binary mode.
 func (c *DataChannel) Send(data []byte) {
+	c.sendInternal(data, true)
+}
+
+// SendText sends a message over the DataChannel in text mode.
+func (c *DataChannel) SendText(data []byte) {
```

Comment:
I would think the signature would be `SendText(data string)` and then call `sendInternal([]byte(data), false)`.